operator/src/test/java/org/keycloak/operator/testsuite/unit/CRSerializationTest.java

```diff
@@ -36,4 +42,22 @@ public void testDeserialization() {
         assertTrue(keycloak.getSpec().isDisableDefaultIngress());
     }
 
-}
+    @Test
+    public void featureSpecificationDeserialization(){
```

Comment:
I think we could replace this by a simple unit test testing that we actually pass the values as correct env vars?

Reply:
I think it's okay to have it here as it's a really fast test, and we can quickly react to the problem with the serialization of the properties. It'll be useful when the new first-class citizen fields are added. Is it ok to keep it?

Reply:
Ok 👍